web_src/css/repository-issue-label.css

```diff
@@ -0,0 +1,44 @@
+.issue-label-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.issue-label-list .item {
+  border-bottom: 1px solid var(--color-secondary);
+  display: flex;
+  padding: 1em 0;
+  margin: 0;
+}
+
+.issue-label-list .item:first-child {
+  padding-top: 0;
+}
+
+.issue-label-list .item:last-child {
+  border-bottom: none;
+  padding-bottom: 0;
+}
+
+.issue-label-list .item .label-title {
+  width: 33%;
+}
+
+.issue-label-list .item .label-issues {
```

Comment:
```suggestion
.issue-label-list .item .label-title, .issue-label-list .item .label-issues {
  width: 33%;
}
```

Reply:
It's not worthy.

The 33% is not a final value , maybe in the future some people would like 40 40 20, etc.